integr8ly/integreatly-operator
# creating solution explorer manifest

create manifest configmap for solution explorer

## Verification
- Create test namespace
- Create installation CR (selfSignedCerts: true)
- Run this code in that namespace
- Once solution explorer goes to phase complete see a manifest secret in the solution explorer namespace containing the values of all completed products.
